### PR TITLE
Feat/um 197 hook up teams

### PR DESCRIPTION
**Pull request by briancao**
Hook up teams UI.

### CODE REVIEW

File: db/postgresql/schema.prisma [hunk 1]
```diff
@@ -118,16 +115,14 @@ model TeamUser {
...
   id        String    @id() @unique() @map("team_website_id") @db.Uuid
   teamId    String    @map("team_id") @db.Uuid
-  userId    String    @map("user_id") @db.Uuid
   websiteId String    @map("website_id") @db.Uuid
   createdAt DateTime? @default(now()) @map("created_at") @db.Timestamptz(6)
 
   team    Team    @relation(fields: [teamId], references: [id])
-  user    User    @relation(fields: [userId], references: [id])
   website Website @relation(fields: [websiteId], references: [id])
+  userId  String? @db.Uuid
```
Comment (mikecao) [inline db/postgresql/schema.prisma:123]: Why is this added?

File: lib/auth.ts [hunk 2]
```diff
@@ -55,6 +56,12 @@ export async function canViewWebsite({ user, shareToken }: Auth, websiteId: stri
     return true;
   }
 
+  const teamWebsite = await getTeamWebsiteByWebsiteIdUserId(websiteId, user.id);
```
Comment (mikecao) [inline lib/auth.ts:59]: This is a goofy name.

**Review by mikecao — CHANGES_REQUESTED** [review]

File: db/postgresql/schema.prisma (hunk 1 above)
Comment (briancao) [reply db/postgresql/schema.prisma:123]: Prisma auto-added after i left something else behind. removed.
